spec/features/has_many_spec.rb

```diff
@@ -19,6 +19,16 @@
     expect(page).to have_content("John Doe")
   end
 
+  scenario "pagination" do
+    FactoryBot.create_list(:student, 10, school: school)
+
+    visit admin_school_path(school)
+
+    expect(page.html).to include("<nav class=\"pagination\" role=\"navigation\" aria-label=\"pager\">")
+    expect(page.html).to include("Next &rsaquo;")
+    expect(page.html).to include("Last &raquo;")
```

Comment:
For these, we can do something like:

```suggestion
    expect(page).to have_content("Last »")
```